spec.md

```diff
@@ -145,6 +145,18 @@ A GET request to an existing blob URL MUST provide the expected blob, with a res
 
 If the blob is not found in the registry, the response code MUST be `404 Not Found`.
 
+##### Checking if content exists in the registry
+
+In order to verify that a repository contains a given manifest or blob, make a `HEAD` request to a URL in the following form:
+
+`/v2/<name>/manifests/<reference>` <sup>[end-12](#endpoints)</sup> (for manifests), or
+
+`/v2/<name>/blobs/<digest>` <sup>[end-11](#endpoints)</sup> (for blobs).
+
+A HEAD request to an existing blob or manifest URL MUST return `200 OK`.
```

Comment:
If this is meant to be returned as body, that would be in conflict with the HTTP specs (see the section I quoted); the body `MUST` be omitted

Reply:
(reading from my phone; I'll read up tomorrow when I'm back at my computer)